e2e/cypress/support/api/user.js

```diff
@@ -256,3 +256,22 @@ Cypress.Commands.add('apiPromoteGuestToUser', (userId) => {
         return cy.apiGetUserById(userId);
     });
 });
+
+/**
+ * Verify a user email via API
+ * @param {String} userId - ID of user of email to verify
+ */
+Cypress.Commands.add('apiVerifyUserEmailById', (userId) => {
```

Comment:
Great @josephk96! I missed to mention to add type as well at `user.d.ts` so this will show up on `cy` autocomplete/intellisense. Should be included at https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api/user.d.ts